src/App/Fossa/Lernie/Types.hs

```diff
@@ -22,6 +24,33 @@ import GHC.Generics (Generic)
 import Path (Abs, Dir, Path)
 import Srclib.Types (LicenseSourceUnit)
 
+data GrepOptions = GrepOptions
+  { customLicenseSearch :: [GrepEntry]
+  , keywordSearch :: [GrepEntry]
+  , ignoreOrgWideCustomLicenseScanConfigs :: Bool
```

Comment:
[Optional] I'd recommend making this an enum as well. Though I don't know what consumers of its JSON format expect so it might be more trouble than it's worth.